Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp

```diff
@@ -670,6 +674,13 @@
       ui.reset_status();
       ui.release();
     #endif
+    #ifdef Z_PROBE_END_SCRIPT
+      if (DEBUGGING(LEVELING)) DEBUG_ECHOLNPAIR("Z Probe End Script: ", Z_PROBE_END_SCRIPT);
+      if (probe_deployed) {
+        planner.synchronize();
+        queue.enqueue_now_P(PSTR(Z_PROBE_END_SCRIPT));
```

Comment:
Please test to make sure this can’t lead to an infinite loop. The command queue cannot be emptied while inside of a G-code command. Therefore if the buffer is full or you try to enqueue more commands than the buffer can hold while inside of a G-code handler the buffer will never get drained and this method will never return.

Reply:
![B6E34588-BB9A-45DC-9F5C-456313D756B0](https://user-images.githubusercontent.com/698003/60283101-0967e700-98ce-11e9-9392-e36e746b0cb3.jpeg)

Reply:
Inside of G-codes you must use `process_subcommands_now_P` instead.

Reply:
Same applied to the other PR as well.